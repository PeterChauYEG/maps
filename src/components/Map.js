import React from 'react'
import ReactMapGL from 'react-map-gl'

// env var
const MAPBOX_ACCESS_TOKEN = process.env.REACT_APP_MAPBOX_ACCESS_TOKEN

const Map = (props) => {
  const {
    map,
    onChangeViewport
  } = props

  return (
    // <ReactMapGL
    //   { ...map }
    //   showZoomControls={true}
    //   width={500}
    //   height={500}
    //   // mapStyle={mapStyle}
    //   mapboxApiAccessToken={MAPBOX_ACCESS_TOKEN}
    //   onChangeViewport={onChangeViewport}
    // />
      <ReactMapGL
        width={400}
        height={400}
        latitude={37.7577}
        longitude={-122.4376}
        zoom={8}
        mapboxApiAccessToken={MAPBOX_ACCESS_TOKEN}
        onViewportChange={(viewport) => {
          const {width, height, latitude, longitude, zoom} = viewport;
          // Optionally call `setState` and use the state to update the map.
        }}
      />
  );
};

export default Map
