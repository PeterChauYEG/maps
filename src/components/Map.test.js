import React from 'react'
import renderer from 'react-test-renderer'
import Map from './Map'

it('renders without crashing', () => {
  const tree = renderer
    .create(<Map />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
