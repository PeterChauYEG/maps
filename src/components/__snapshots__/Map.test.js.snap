// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Map"
>
  <h1>
    Maps
  </h1>
</div>
`;
